backend/siarnaq/api/compete/views.py

```diff
@@ -412,50 +487,48 @@ def scrimmage(self, request, pk=None, *, episode_id):
     )
     def historical_rating(self, request, pk=None, *, episode_id):
         """List the historical ratings of a list of teams."""
-        queryset = Match.objects.all().filter(tournament_round__isnull=True)
-
         team_ids = self.request.query_params.getlist("team_ids")
         if team_ids is not None and len(team_ids) > 0:
             team_ids = {parse_int(team_id) for team_id in team_ids}
```

Comment:
I think we should change this endpoint to take in a singular team id now? The only reason this was changed was to accommodate the top 10 list? I believe

Reply:
We never fetch them concurrently so i think this is ok. We only fetch for one episode at a time